### DISCUSSION TITLE
SQL Server 14.0.100 connection error

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#6796] [General]
Hi,

I'm trying to use serene with SQL Server; 
I changed default connection string in appsettings.json in this way:
```
  "Data": {
    "Default": {
      //"ConnectionString": "Server=(localdb)\\MsSqlLocalDB;Database=Serene13Giu_Default_v1;Integrated Security=true;Encrypt=false;TrustServerCertificate=true",
      "ConnectionString": "Persist Security Info=True;Data Source=.\\SQLExpress;Initial Catalog=Serene13Giu;Integrated Security=true;connect timeout=100;TrustServerCertificate=false",
      "ProviderName": "System.Data.SqlClient"
    }
  },
```
When I run the application, this error occurs:

`Microsoft.Data.SqlClient.SqlException: 'A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - Certificate chain issued by a Certification Authority not available in the local directory.)'`

Any idea?

Thank you,
Elena

**Comment by wezmag** [comment]
Check out this doc. 
https://github.com/serenity-is/Serenity/blob/29b3fcdf1ad3b2171df0f562e7aaf472b3425a16/CHANGELOG.md#650-2023-01-26

**Reply by elenaaralla** [reply]
@wezmag: thank you!

**Comment by elenaaralla** [comment]
Hi,

with your suggestion the connection error disappeared but a new error appear now:
**_The object name 'Languages' is invalid_**

An unhandled exception occurred while processing the request.
SqlException: Il nome di oggetto 'Languages' non è valido. (The object name 'Languages' is invalid)

I only created the project, corrected connection error setting _TrustServerCertificate=true_ and runned it.

I'm using VS 2022
SQL Server SQL Server 14.0.100

See image attached
![Schermata 2023-06-14 alle 09 03 28](https://github.com/serenity-is/Serenity/assets/14870650/82fcf473-3125-4f7f-afd6-b3c8be13826e)

Thank you

**Reply by elenaaralla** [reply]
Ok, i found the problem.

1) I've removed the database (that I created before running the project)
2) I've commented the "safety check to ensure that..." code line in DataMigration.cs

Is there a documentation or "how to" where this 2 steps are described (and I've missed)?

Thank you,
Elena

**Reply by wezmag** [reply]
Maybe here > https://serenity.is/docs/tutorials/movies/creating_movie_table
I am not sure if you have gone through the tutorials before. They cover all the basics about Serenity Framework.
I highly recommend going through them at least once.

**Reply by elenaaralla** [reply]
Hi @wezmag,

thanks.
I had followed the tutorial, using serene default db, and I missed the step related to safety check on database name.

Have a nice day.

**Comment by Nelly1996** [comment]
I had a similar error thank you!!
